scripts/benchmark_all.sh

```diff
@@ -12,7 +12,17 @@ then
 fi
 
 benchmarks='hyriseBenchmarkTPCH hyriseBenchmarkTPCDS hyriseBenchmarkTPCC hyriseBenchmarkJoinOrder'
-num_mt_clients=50
+warmup_seconds=2
+runs=100
```

Comment:
Change in line 7: "Running it takes a few hours."

Reply:
I guess that will not change.

Reply:
Well, currently it is "Running it takes more than an hour.", which is an understatement. 😄